EnglishFilter/sections/general_extensions.txt

```diff
@@ -168,7 +168,7 @@ streamtape.com#%#//scriptlet("abort-current-inline-script", "Math", "/emZnbG9hZG
 lookmovie.ag#%#//scriptlet("abort-on-property-write", "zfgproxyhttp")
 ! #%#//scriptlet("prevent-setInterval", "[$."), #%#//scriptlet("prevent-setTimeout", "[$."), #%#//scriptlet("prevent-addEventListener", "", "[$.")
 manga-raw.club#%#//scriptlet("prevent-setInterval", "[$.")
-kickass.ws,kickass.love,kissanime.nz,animekisa.tv,repackov.com#%#//scriptlet("prevent-setTimeout", "[$.")
+123movieslinks.live,kickass.ws,kickass.love,kissanime.nz,animekisa.tv,repackov.com#%#//scriptlet("prevent-setTimeout", "[$.")
```

Comment:
I meant another rule, but this one is also work. Ok.

Reply:
Ok thanks.